src/panels/lovelace/entity-rows/hui-input-number-entity-row.js

```diff
@@ -103,6 +105,11 @@ class HuiInputNumberEntityRow extends mixinBehaviors(
       },
       _step: Number,
       _value: Number,
+      _rtl: {
+        type: String,
+        reflectToAttribute: true,
```

Comment:
why reflect to attribute

Reply:
Because that's what Polymer docs says:
    /**
      * NOTE(keanulee): Though :host-context is not universally supported, some pages
      * still rely on paper-slider being flipped when dir="rtl" is set on body. For full
      * compatibility, dir="rtl" must be explicitly set on paper-slider.
      */

Setting style doesn't work. Tried it...

Reply:
You are not using it as selector in CSS now, so you don't need `reflectToAttribute`?

Reply:
Fixed. Now I bothered reading what that meant. Was copy-pasting. Sorry.